### DISCUSSION TITLE
Welcome to lingarr Discussions!

### DISCUSSION THREAD

**Discussion by rowanfuchs** [lingarr-translate/lingarr#18] [Announcements]
👋 **Welcome to Lingarr Community!**

We're using Discussions as a place to connect with other members of our Lingarr community. We hope that you:

- Ask questions about Lingarr's features and capabilities.
- Share ideas for improving subtitle translation and Feature requests.
- Engage with other Lingarr users and developers.
- Share your experiences with both SaaS and locally hosted translation services.
- Welcome others and are open-minded. Remember that this is a community we build together to make subtitle translation easier and more accessible 💪

To get started, comment below with an introduction of yourself and tell us about how you use Lingarr for your subtitle translation needs. Are you using the manual translation feature, the automatic translation, or both? Do you prefer the SaaS option or the locally hosted service? We'd love to hear about your experience!

**Comment by Fahad-Beta** [comment]
First of all thank you for this amazing idea even though I didn't get the chance to try it since I'm in a seedbox but hopefully this will see the light into the arr family and will be able available as a basic application.

Do you think it's possible for your app to integrate with Bazarr? It feels like the two are fit for each other. Bazarr really need ai translator other than whisper.

**Comment by rowanfuchs** [comment]
Thank you! the idea is there but it depends on Bazarr's plans on development and what they plan with their translation feature.

**Reply by Fahad-Beta** [reply]
It's nice to hear, based on my quick talk with bazarr developer in discord they are open for it and I was requested to submit a feature request which I already did. We hope to see it someday.

I believe we will see it happens faster if you both reach out. Bazarr developers are cool for any good plans.

**Reply by rowanfuchs** [reply]
thats awesome, I've been in contact but I haven't continued conversations. I will pick it up once we are closing in on a stable release.
